deploy/olm-catalog/performance-addon-operator/4.6.0/performance.openshift.io_performanceprofiles_crd.yaml

```diff
@@ -190,7 +190,7 @@ spec:
             type: object
         type: object
     served: true
-    storage: true
+    storage: false
```

Comment:
That's a bug

Reply:
False alarm, that's actually ok, V1 is not stored anymore, V2 is stored now, sorry for the confusion.